assignment_4.0.py

```diff
@@ -0,0 +1,216 @@
+#Birth
+
+class Child:
+
+    def __init__(self, name, gender,weight):
+        self.name = name
+        self.gender = gender
+        self.weight = weight
+
+    def birth_details(self):
+        print("A child emerges the earth.")
+        print("Name:",self.name)
+        print("Gender:",self.gender)
+        print("Weight:",self.weight, "kg" )
+
+    
+child_0 = Child("Ziyan","Male",4.0)
+child_0.birth_details()
+
+child_1 = Child("Ziya","Female",3.4)
+child_1.birth_details()
+
+child_2 = Child("Ram","Male",4.5)
+child_2.birth_details()
+
+
+
+class ChildToSchool:
+
+    def __init__(self,name,age,grade,sex):
+        self.name = name
+        self.age = age
+        self.grade = grade
+        self.sex = sex
+
+    def can_appear_for_admission(self,eligibility):
+        if self.age >= eligibility["Age_Bar"]:
+            return True
+        else:
+            return False
+
+
+
+class SchoolSystem:
+
+    def __init__(self):
+        self.students = {}
+        self.students_joined = {}
+        self.students_rejected = {}
+        self.students_on_hold = {}
+
+    def add_student(self,student):
+        self.students[student.name] = student
+
+    def register_student_for_admission(self,student):
+        self.add_student(student)
+
+    def process_admission(self,school_name,eligibility):
+        eligible_students = []
+        for student_name, student in self.students.items():
+            if eligible_students.can_appear_for_admission(eligibility):
```

Comment:
This will still fail. Did you run the code? eligible students is list. list cannot have can_appear_for_admission function.

Reply:
i made the changes now